src/pages/Zomunk.html

```diff
@@ -66,4 +66,5 @@ <h2 class="mt-5">Here is how it works in short</h2>
   <!-- Bootstrap JS -->
```

Comment:
Automated Code Review Suggestion:

The issue is the trailing whitespace on line 67 and the lack of a newline at the end of the file.  While not strictly an error, these can cause unnecessary diff noise and some linters complain.  This cleans up the file.

```suggestion
</html>

```